conda/cli/main_info.py

```diff
@@ -499,43 +499,36 @@ def _json_all_component(self) -> dict[str, Any]:
         return self._info_dict
 
 
-def get_info_components(args: Namespace, context: Context) -> set[InfoComponents]:
+def get_info_components(args: Namespace, context: Context) -> Iterable[InfoComponents]:
```

Comment:
I'm curious: is this type of return type change an API breaking one? If so, do we need to follow deprecation cycles?

Reply:
Wait did we want to do this deprecation?